cdk/lib/payment-api.ts

```diff
@@ -131,13 +131,20 @@ export class PaymentApi extends GuStack {
               `arn:aws:ssm:${this.region}:${this.account}:parameter/${app}/*`,
             ],
           }),
-          new GuAllowPolicy(this, "SqsMessages", {
+          new GuAllowPolicy(this, "EmailSqsMessages", {
             actions: ["sqs:GetQueueUrl", "sqs:SendMessage"],
             resources:
               this.stage === "PROD"
                 ? [emailSqsCodeArn.valueAsString, emailSqsProdArn.valueAsString]
                 : [emailSqsCodeArn.valueAsString],
           }),
+          new GuAllowPolicy(this, "SoftOptInsSqsMessages", {
+            actions: ["sqs:GetQueueUrl", "sqs:SendMessage"],
+            resources:
+              this.stage === "PROD"
+                ? [`arn:aws:sqs:${this.region}:${this.account}:soft-opt-in-consent-setter-queue-PROD`,]
```

Comment:
is it better to import an exported ARN instead?

Reply:
I don't think there's anything wrong with this, and I _think_ it's fairly idiomatic GuCDK use.